ads/pubmine.js

```diff
@@ -46,6 +46,7 @@ function initMasterFrame(data, global) {
     tn: 'amp',
     amp: true,
     consent: consent ? 1 : 0,
+    siteid: Number(data['siteid']) || undefined,
```

Comment:
Is there any particular reason this needs to be a number? I can't recall if there are any situations where it's not, but I recall at some point hearing that the site identifier wasn't necessarily required to be a number in the back end.

I guess what I'm trying to ask: is it possibly we are over-limiting ourselves here in a way that might be challenging if we add letters to the site id at some future point?

Reply:
For now we use siteid as a number on both js and server sides. It can be a string theoretically, but it follows with lots of work to migrate to this approach

Reply:
Okay, let's go go ahead as-is. We can always update here later.